moabb/evaluations/splitters.py

```diff
@@ -101,3 +109,112 @@ def split(self, y, metadata):
                 for train_ix, test_ix in splitter.split(indices, y_session):
 
                     yield indices[train_ix], indices[test_ix]
+
+
+class CrossSessionSplitter(BaseCrossValidator):
+    """Data splitter for cross session evaluation.
+
+    Cross-session evaluation uses a Leave-One-Session-Out cross-validation to
+    evaluate performance across sessions, but for a single subject. This splitter
+    assumes that all data from all subjects is already known and loaded.
```

Comment:
I think we should clearly explain the difference/similarity between `CrossSessionSplitter` and `CrossSessionEvaluation` in both of their docsrtings (same for the within-session classes). It could be quite confusing to new users who want to do a cross-session evaluation as there are now two classes with that name

Reply:
Fair point!